src/dlls/mscorrc/mscorrc.rc

```diff
@@ -987,8 +987,8 @@ BEGIN
     IDS_EE_NDIRECT_BADNATL_CALLCONV         "Invalid PInvoke or UnmanagedFunctionPointer calling convention."
     IDS_EE_NDIRECT_BADNATL_VARARGS_CALLCONV "Invalid PInvoke calling convention. Vararg functions must use the cdecl calling convention."
     IDS_EE_NDIRECT_BADNATL_THISCALL         "Invalid PInvoke calling convention. Thiscall requires that the first parameter is present and can be enregistered."
-    IDS_EE_NDIRECT_LOADLIB                  "Unable to load DLL '%1': %2"
-    IDS_EE_NDIRECT_GETPROCADDRESS           "Unable to find an entry point named '%2' in DLL '%1'."
+    IDS_EE_NDIRECT_LOADLIB                  "Unable to load shared library '%1': %2"
```

Comment:
It is pretty unusual to use "shared library" name on Windows. If we want to use "shared library" for Unix, Windows version of the error message should keep "DLL".